Godeps/Godeps.json

```diff
@@ -24,8 +24,8 @@
 		},
 		{
 			"ImportPath": "github.com/urfave/cli",
-			"Comment": "v1.17.0-57-gedb24d0",
-			"Rev": "edb24d02aa3cea2319c33f2836d4a5133907fe4c"
+			"Comment": "v1.18.0-47-g168c954",
+			"Rev": "168c95418e66e019fe17b8f4f5c45aa62ff80e23"
```

Comment:
Can we bump this to v1.18.0 on the nose (urfave/cli@1efa31f)?  It [includes the features you want][1].

[1]: https://github.com/urfave/cli/blob/v1.18.0/CHANGELOG.md#added